recipe/meta.yaml

```
@@ -1,52 +1,65 @@
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_beat.py " %}
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_control.py " %}
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_worker.py " %}
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_daemonization.py " %}
+# celery.exceptions.SecurityError: Running a worker with superuser privileges
+{% set u_tests_to_skip = " test_check_privileges " %}
+# Failed: DID NOT RAISE any of (<class 'ValueError'>, <class 'Failed'>)
+{% set u_tests_to_skip = u_tests_to_skip + "or test_pytest_celery_marker_registration " %}
+# AssertionError: assert 'worker1 [2024-01-01 12:00:00]' in 'worker1 [2024-01-01 10:00:00] task received mytask(abc) args=(1,) kwargs={} \n'
+{% set u_tests_to_skip = u_tests_to_skip + "or test_on_event_task_received " %}
+# AssertionError: assert 'worker1 [2024-01-01 12:00:00]' in 'worker1 [2024-01-01 10:00:00] started: foo=bar\n'
+{% set u_tests_to_skip = u_tests_to_skip + "or test_on_event_non_task " %}
 
 package:
-    name: {{ name|lower }}
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: f4efebe6f8629b0da2b8e529424de376494f5b7a743c321c8a2ddc2b1414921c
-    patches:
-        - 0001-skip-a-test.patch
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c972ae7968c2b5281227f01c3a3f984037d21c5129d07bf3550cc2afc6b10a5
 
 build:
-    noarch: python
-    number: 0
-    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-    entry_points:
-        - celery = celery.__main__:main
+  skip: true  # [py<38]
+  entry_points:
+    - celery = celery.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
-    build:
+    host:
         - python
         - pip
-        - patch     # [not win]
-        - m2-patch  # [win]
-
+        - wheel >=0.33.1
+        - setuptools >=40.8.0
     run:
         - python
-        - click-didyoumean
-        - click-repl
-        - click-plugins
-        - pytz
-        - billiard>=3.6.3,<4.0
-        - kombu>=4.6.10,<4.7
-        - future >=0.18.0
-        - vine 1.3.0.*
-        - sqlalchemy
-        - cryptography
-    run_constrained:
-        - sphinx >=2.0
-
+        - billiard >=4.2.1,<5.0
+        - kombu >=5.5.2,<5.6
+        - vine >=5.1.0,<6.0
+        - click >=8.1.2,<9.0
+        - click-didyoumean >=0.3.0
+        - click-repl >=0.2.0
+        - click-plugins >=1.1.1
+        - backports.zoneinfo >=0.2.1  # [py<39]
+        - python-dateutil >=2.8.2
 test:
     requires:
-        - case >=1.3.1
-        - pytest >=4.5.0,<5.3.5
+        - pytest 8.4.1 # 8.3.5 version is required but we don't have this exact version, the newer one is compatible
```

This should cover it then 😄 

```suggestion
        - pytest >=8.3.5
```